utils/preprocessing.py

```diff
@@ -63,6 +64,45 @@ def preprocess_ocean_warming(df: pd.DataFrame) -> pd.DataFrame:
     return df
 
 
+def preprocess_world_employment(df:pd.DataFrame) -> pd.DataFrame:
+    df = df.groupby(by=['TIME']).mean().reset_index()
+    df = df[df['TIME'] > 1964]
+    df = df.rename(columns = {'TIME':'year'})
+    df.insert(loc=1, column='month', value=12)
+    df.insert(loc=2, column='day', value=31)
+    df.insert(loc=0, column='date', value=pd.to_datetime(df[['year','month','day']]))
+    df.drop(['year','month','day'], axis=1, inplace=True)
+    df.rename(columns= {'Value':'world_employment_rate'}, inplace=True)
+    return df
+
+def preprocess_energy_substitution(df:pd.DataFrame) -> pd.DataFrame:
+    column_list = ['Other renewables (TWh, substituted energy)',
+                   'Biofuels (TWh, substituted energy)',
+                    'Solar (TWh, substituted energy)',
+                   'Wind (TWh, substituted energy)',
+                   'Hydropower (TWh, substituted energy)',
+                   'Nuclear (TWh, substituted energy)',
+                   'Gas (TWh, substituted energy)',
+                   'Oil (TWh, substituted energy)',
+                   'Coal (TWh, substituted energy)',
+                   'Traditional biomass (TWh, substituted energy)']
+    df['global_energy_substitution'] = df[column_list].sum(axis=1)
```

Comment:
A more efficient way with way fewer lines of code would be: df.filter(like='substituted energy', axis=1)